### ISSUE TITLE
Assertion failed: (sidx>=0), function ParseFromRPN, file parser/mpParserBase.cpp, line 1103.

### ISSUE THREAD

**Issue by nasailja** [beltoforion/muparserx#55]
When parsing this I get the assertion above with muparserx from yesterday's git: `{0,{0}}*{0,{0}}*{,{0}}*{0,{0}0,{0}}*{0,{0}}*{,{0}}*{0}*{000}`, found with http://lcamtuf.coredump.cx/afl/

Parsing program:
```
#include "iostream"
#include "string"
#include "mpParser.h"

int main()
{
	std::string expression;
	std::cin >> expression;

	mup::ParserX parser(mup::pckCOMMON | mup::pckNON_COMPLEX | mup::pckMATRIX | mup::pckUNIT);
	mup::Value value1{0.0};
	mup::Variable variable1{&value1};
	parser.DefineVar("v1", variable1);
	parser.SetExpr(expression);

	try {
		double result = parser.Eval().GetFloat();
	} catch (...) {
	}

	return 0;
}
```

**Comment by nasailja** [comment]
Another expression, also from afl: `{{,0k*00,0k*0}0}`

**Comment by beltoforion** [comment]
Fixed in V4.0.1
